### ISSUE TITLE
Dropdown task labels aren't clickable/tappable

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/front-end-monorepo#5706]
## Package
_Choose from the list:_
- [ ] app-content-pages
- [ ] app-project
- [x] lib-classifier
- [ ] lib-panoptes-js
- [ ] lib-react-components
- [ ] unknown

## Describe the bug
When I click or tap on the text task instruction, the dropdown menu of suggestions opens and I can select an option.

https://github.com/zooniverse/front-end-monorepo/assets/59547/782424d7-2b34-4c82-87d8-93fb2df00852

When I click or tap the instruction for a dropdown menu, nothing happens. I have to click inside the control itself, in order to activate it.

https://github.com/zooniverse/front-end-monorepo/assets/59547/b3f8a672-37c7-4a07-875b-0ceaf518d0a6

## To Reproduce
Click or tap the task instruction for [the dropdown task](https://zooniverse.github.io/front-end-monorepo/@zooniverse/classifier/index.html?path=/story/tasks-simple-dropdown--short-options) in the classifier storybook.

## Expected behavior
UX should be consistent between tasks. Form controls should also have accessible labels. Clicking the label for a dropdown task should open the menu and focus the control.

**Comment by Saicbmm** [comment]
After reviewing the source code for the link provided, it seems that the "Select your favourite colour" label is in a <p> tag. One way to implement the suggested behavior is by adding a <button> tag inside of the <p> tag so that when the user presses it, the dropdown list pops up.
